distmv/statdist.go

```diff
@@ -0,0 +1,163 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
```

Comment:
In the past, we've had the copyright year as the year the code was written, and not the year the code was submitted. This was written last year, so should these years stay?